www/src/components/navigation.js

```diff
@@ -3,10 +3,9 @@ import { jsx } from "theme-ui"
 import { useColorMode } from "theme-ui"
 import GithubIcon from "react-icons/lib/go/mark-github"
 import TwitterIcon from "react-icons/lib/fa/twitter"
-
 import Link from "../components/localized-link"
 import SearchForm from "./search-form"
-import DiscordIcon from "../components/discord"
+import {FaDiscord}  from "react-icons/fa";
```

Comment:
Could you do

```
import DiscordIcon from "react-icons/fa/discord"
```

This is what we're doing with the other icons.

Reply:
sure